xbmc/windowing/gbm/drm/DRMObject.cpp

```diff
@@ -56,9 +55,8 @@ std::string CDRMObject::GetPropertyName(uint32_t propertyId) const
 
 uint32_t CDRMObject::GetPropertyId(const char* name) const
 {
-  auto property = std::find_if(m_propsInfo.begin(), m_propsInfo.end(), [&name](auto& prop) {
-    return StringUtils::EqualsNoCase(prop->name, name);
-  });
+  auto property = std::find_if(m_propsInfo.begin(), m_propsInfo.end(),
+                               [&name](auto& prop) { return prop->name == name; });
```

Comment:
I assume the change from case insensitive to case sensitive comparison is fine?

Reply:
Nice catch, I'll make a note of that in the OP.

reason for the change is here https://github.com/xbmc/xbmc/pull/18567#issuecomment-709874406